texar/modules/connectors/__init__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Modules of texar library connectors.
+"""
+
+# pylint: disable=wildcard-import
```

Comment:
This directive is redundant and can be removed. In general we don't use file-level `pylint: disable` directives unless the same warning is triggered multiple times in the file.